views/updateBrowser/updateBrowser.html

```diff
@@ -0,0 +1,18 @@
+<div class="jw-cover jw-modal ng-cloak">
+    <div class="jw-modal-container">
+        <div class="jw-modal-image">
+            <img class="jw-header-logo" ng-src="{{ ::vm.config.assets.banner }}" alt="logo"/>
+        </div>
+        <div class="jw-modal-message jw-modal-message-title">
+            Your browser is out-of-date.
```

Comment:
Some of the features used by JW Showcase aren't support by your current browser.

Reply:
agree, and done